daemon/daemon_unix.go

```diff
@@ -177,6 +177,10 @@ func parseSecurityOpt(container *container.Container, config *containertypes.Hos
 		if strings.Contains(opt, "=") {
 			con = strings.SplitN(opt, "=", 2)
 		} else if strings.Contains(opt, ":") {
+			// Note: keeping this to be retro-compatible. From API 1.27 and later, a validation is
+			// done upfront and this will fail early.
+			// But as long as we support API versions lower than 1.27, we should still be
+			// able to handle `:` at this point of the code
```

Comment:
IIUC, these options are read from the container's HostConfig, which means that a container _created_ with an old API version still would have `:`. Should we

- _migrate_ the options in the container-config?
- start thinking of a more standardized way of storing these options?